src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1590,6 +1612,14 @@ private void loanHasExpectedProperties(JsonObject loan) {
     hasProperty("status", loan, "loan");
     hasProperty("action", loan, "loan");
     hasProperty("item", loan, "loan");
+    hasProperty(LoanProperties.BORROWER, loan, "loan");
```

Comment:
This method was originally intended for checking properties of loans in collection methods.

Your change has made me think that maybe it could be expanded to a general check, thank you.

Reply:
It is indeed a general check, and all existing methods that use loanHasExpectedProperties() will now expect the borrower fields in a loan.

Reply:
This method only checks that the properties are present. It does not check they are populated correctly. 

The existing asserts in `canCreateALoan` include checks for the specific values of properties. I believe that could be useful for these properties as well. 

What do you think?

Reply:
Thank you for your input. I think it does make sence, so as far as the borrower fiels go,  I have refactored the tests so that the values of the properties are now checked as well as the presence of those properties.

Reply:
I had a different approach in mind for how this could be done (with explicit examples in the specific tests).

I'm not sure there is much value in me delaying this change any further, by asking for further changes.

I do think there needs to be a test for a closed loan without a user (an anonymized loan), for demonstrating that the borrower information is not present.

Reply:
The scenario to demonstrate that there is no borrower information for an anonymized loan has been implemented and covered with tests.

Reply:
Thank you for adding the tests for a loan without a user